### DISCUSSION TITLE
ValueError - insufficient funds for gas*price

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#1019] [General]
I am doing the python/brownie course from Patrick (free code camp channel on Youtube). I am trying to run the following test on mainnet-fork-dev, and I am getting an error "ValueError: insufficient funds for gas * price + value". I believe this means I need to have some eth in my forked account to pay for gas, but how do I get that? Do I have to actually deposit real eth into my account (since this is a fork of mainnet)? We used alchemy for adding mainnet-fork-dev as a network on brownie.

When I send "tx = fund_me.fund({"from": account, "value": entrance_fee})" could I just add "gas": 100, "gas_price": 1 or something to simulate paying gas?

``` solidity
`def test_can_fund_and_withdraw():
    account = get_account()
    fund_me = deploy_fund_me()
    entrance_fee = fund_me.getEntranceFee()
    tx = fund_me.fund({"from": account, "value": entrance_fee})
    tx.wait(1)
    assert fund_me.s_addressToAmountFunded(account.address) == entrance_fee
    tx2 = fund_me.withdraw({"from": account})
    tx2.wait(1)
    assert fund_me.s_addressToAmountFunded(account.address) == 0`

**Comment by 0xusmanf** [comment]
Try using Sepolia.

**Reply by 0xwtfwtf** [reply]
Oh yeah it works on development and on sepolia...I was just trying to get it to work on mainnet-fork-dev too because it is part of the course

**Reply by 0xusmanf** [reply]
In order to run it on mainnet fork you need real ETH in your account and you can't just simulate paying gas without it.
